content/50_deploying_argocd_to_eks/30_declare_argocd_helm_chart.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "Declare an ArgoCD Helm Chart Object"
+chapter = false
+weight = 30
++++
+
+You'll now declare a [helm chart](https://kubernetes.io/blog/2016/10/helm-charts-making-it-simple-to-package-and-deploy-apps-on-kubernetes/),
+which deploys all of the components that is required to deploy ArgoCD.
+
+Next, let's declare the helm chart:
+
+```typescript
+const argocd = new k8s.helm.v2.Chart("argocd",
+    {
+        namespace: ns.metadata.name,
+        chart: "argo-cd",
+        fetchOpts: { repo: "https://argoproj.github.io/argo-helm" },
+        values: {
+            installCRDs: false,
+            server: {
+                service: {
+                    type: 'LoadBalancer',
+                },
+            }
+        },
+        // The helm chart is using a deprecated apiVersion,
+        // So let's transform it
+        transformations: [
+            (obj: any) => {
+                if (obj.apiVersion == "extensions/v1beta1")  {
+                    obj.apiVersion = "networking.k8s.io/v1beta1"
+                }
+            },
+        ],
+    },
+    { providers: { kubernetes: provider }},
+);
+```
+
+Lastly, we need to get the Url of our ArgoCD deployment:
```

Comment:
consider adding a note as to why we are doing this:

e.g.
"The ArgoCRD deployed by the Helm chart is outside of the management of Pulumi. However, Pulumi can still query the Helm chart resource to extract resource properties."

Reply:
love that!